app/javascript/App/bundles/NewsfeedPage/NewsBody.tsx

```diff
@@ -104,12 +104,12 @@ export default class NewsBody extends React.Component<Props, State> {
             rel="nofollow"
             className="break-all"
           >
-            <Icon name="link" className="mr1 f7" regular={true} />
+            <Icon name="link" className="mr1 f7" styleType="regular" />
```

Comment:
Given the icon component logic, it seems you might as well remove styleType="regular"